erpnext/accounts/doctype/payment_request/payment_request.py

```diff
@@ -214,9 +214,11 @@ def set_as_cancelled(self):
 
 	def check_if_payment_entry_exists(self):
 		if self.status == "Paid":
-			payment_entry = frappe.db.sql_list("""select parent from `tabPayment Entry Reference`
-				where reference_name=%s""", self.reference_name)
-			if payment_entry:
+			payment_entry = frappe.get_all("Payment Entry Reference",
+				filters={"reference_name": self.reference_name, "docstatus": ["<", 2]},
+				fields=["distinct(parent)"])
+
+			if any(payment_entry):
```

Comment:
How's this different than just `if payment_entry`? Asking cause I dunno.

Reply:
Logically, no difference here. But for readability, `any` always takes a list of values and checks all of them for truthy values.